### ISSUE TITLE
Build of falcosecurity/falco:stable is broken because of libctf-nobfd0

### ISSUE THREAD

**Issue by leodido** [falcosecurity/falco#1015]
**Which jobs are failing**:

No test job is failing right now.
Yet the build of `falcosecurity/falco:stable` is broken.

**Which test(s) are failing**:

No tests are failing right now because we do not test (at the moment) the build of docker images.

**Since when has it been failing**:

Since tonight.

**Test link**:

This is the same error for `docker/local` docker image.

See issue #1013 and an online example of that/this error [here](https://circleci.com/gh/falcosecurity/falco/151).

**Reason for failure**:

```
dpkg: warning: downgrading binutils-common:amd64 from 2.33.90.20200122-2 to 2.30-22
(Reading database ... 16224 files and directories currently installed.)
Preparing to unpack binutils-common_2.30-22_amd64.deb ...
Unpacking binutils-common:amd64 (2.30-22) over (2.33.90.20200122-2) ...
dpkg: warning: downgrading binutils-x86-64-linux-gnu from 2.33.90.20200122-2 to 2.30-22
Preparing to unpack binutils-x86-64-linux-gnu_2.30-22_amd64.deb ...
Unpacking binutils-x86-64-linux-gnu (2.30-22) over (2.33.90.20200122-2) ...
dpkg: warning: downgrading binutils from 2.33.90.20200122-2 to 2.30-22
Preparing to unpack binutils_2.30-22_amd64.deb ...
Unpacking binutils (2.30-22) over (2.33.90.20200122-2) ...
dpkg: warning: downgrading libbinutils:amd64 from 2.33.90.20200122-2 to 2.30-22
Preparing to unpack libbinutils_2.30-22_amd64.deb ...
Unpacking libbinutils:amd64 (2.30-22) over (2.33.90.20200122-2) ...
Setting up binutils-common:amd64 (2.30-22) ...
dpkg: dependency problems prevent configuration of libbinutils:amd64:
 libctf-nobfd0:amd64 (2.33.90.20200122-2) breaks libbinutils (<< 2.33.50.20191128-1~) and is installed.
  Version of libbinutils:amd64 to be configured is 2.30-22.

dpkg: error processing package libbinutils:amd64 (--install):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of binutils-x86-64-linux-gnu:
 binutils-x86-64-linux-gnu depends on libbinutils (= 2.30-22); however:
  Package libbinutils:amd64 is not configured yet.

dpkg: error processing package binutils-x86-64-linux-gnu (--install):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of binutils:
 binutils depends on libbinutils (= 2.30-22); however:
  Package libbinutils:amd64 is not configured yet.
 binutils depends on binutils-x86-64-linux-gnu (= 2.30-22); however:
  Package binutils-x86-64-linux-gnu is not configured yet.

dpkg: error processing package binutils (--install):
 dependency problems - leaving unconfigured
Processing triggers for libc-bin (2.29-9) ...
Errors were encountered while processing:
 libbinutils:amd64
 binutils-x86-64-linux-gnu
 binutils
```

**Anything else we need to know**:

This error originates [here in the source code](https://github.com/falcosecurity/falco/blob/32b373aa9a70bb00cefce9e6a866246a72e8159f/docker/stable/Dockerfile#L96-L104).

This is a fairly recent bug..

Another bug report [here](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=949454).

**Comment by leodido** [comment]
/kind bug

Maybe @mstemm already encountered this reecently?

**Comment by leodido** [comment]
I suggest deprecating this image (and also the `docker/local`) for the 1.0.0 since I think this image is not robust enough and Falco cannot deal with this.

A solution could be to switch to ubuntu 18.04 as base image for the docker stable.
Alternatively we'd need to deprecate support for kernels <= 4.16 in the docker image.

/milestone 1.0.0

**Comment by kris-nova** [comment]
I agree with deprecating any images that we aren't using. I know @mfdii did a lot of work with the slim images:

 - [Slim dev](https://github.com/falcosecurity/falco/tree/dev/docker/slim-dev)
 - [Slim stable](https://github.com/falcosecurity/falco/tree/dev/docker/slim-stable)

--- 

Furthermore I opened up #1018 to fix this bug.

If folks are running into this issue today and need a quick fix I [pushed a container image to docker hub](https://hub.docker.com/repository/docker/krisnova/falco-debian) that can be used. 

```
krisnova/falco-debian:stable
```

**Comment by kris-nova** [comment]
Once #1018 is merged this issue can be closed in favor of #1019
